contrib/k8s/dex-worker.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dex
+type: Opaque
+data:
+  key-secrets: ZUhoNGVIaDRlSGg0ZUhoNGVIaDRlSGg0ZUhoNGVIaDRlSGg0ZUhoNGVIZz0= # 32 x's base64 encoded twice.
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: dex
+    role: worker
+  name: dex-worker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: dex
+        role: worker
+    spec:
+      containers:
+          - image: quay.io/coreos/dex
+            name: dex-worker
+            env:
```

Comment:
I wouldn't really worry about that in this PR. Environment variables while not ideal for these kinds of things, this is no worse than the current `cat` approach. 

There is also an open issue for making dex take a file for all of these configuration needs already and fixing up the example k8s manifest to use that in the future can be done in a separate PR once that lands.